apps/nest-server/README.md

```diff
@@ -1,9 +1,31 @@
-# nest-server
+# NestJS Application
 
-This application was generated with [Nx](https://nx.dev).
+This application was generated using [Nx](https://nx.dev).
 
-## Running
+## Running the Application
 
-First run `npm run presto:up` to run a local Presto using `docker-compose` and then `npm run serve nest-server` to serve the application.
+To run this application, follow these steps:
 
-Make a `GET` request to `http://localhost:3000/api/call-centers` to see the response from querying the local Presto using `prestodb-js-client` library.
+1. Start a local Presto instance using `docker-compose` by running the following command:
+
+   ```bash
+   npm run presto:up
+   ```
+
+2. Once the Presto instance is up and running, you can start the Nest Server by running:
+
+   ```bash
+   npm run serve nest-server
+   ```
+
+3. After starting the Nest Server, you can make a `GET` request to the following endpoint to see the response from querying the local Presto container using the `prestodb-js-client` library:
+
+   ```
+   curl -i -H "Accept: application/json" -H "Content-Type: application/json" -X GET http://localhost:3000/api/call-centers | json
```

Comment:
can we remove the `json` part? it's not available everywhere

Reply:
![image](https://github.com/prestodb/presto-js-client/assets/680468/c97da3be-5f89-4232-a028-836a9917fec8)

![image](https://github.com/prestodb/presto-js-client/assets/680468/074ad811-7de5-4b9d-ba56-9110bc225e71)

Reply:
I added it as a dependency, you have to run a `npm i`